src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,50 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100, hidden = false) {
+    this.name = name;
+    this.health = health;
+    this.hidden = hidden;
+    Animal.addAnimal(this);
+  }
+
+  static alive = [];
```

Comment:
```suggestion
Animal.alive = [];
```
lets change it like this move it outside of Animal class